db/migrate/20180109073517_add_current_sign_in_at_to_admins.rb

```diff
@@ -0,0 +1,5 @@
+class AddCurrentSignInAtToAdmins < ActiveRecord::Migration[5.0]
+  def change
+    add_column :admins, :current_sign_in_at, :datetime
```

Comment:
why having this? 🤔  It should be captured by devise in `last_sign_in_at`

Reply:
You also don't use it elsewhere

Reply:
this is requirement from devise's `:trackable`